package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.3",
+    "@types/nodemailer": "^6.4.8",
```

Comment:
I think `types` should be installed as `devDependencies`

Reply:
Done